src/dht/dht.rs

```diff
@@ -839,19 +838,13 @@ impl DHT {
                                 .add_or_update(Node::new(their_id, target), true);
                         }
 
-                        match response_variant {
-                            // Special handling for find_node responses
-                            // Add the nodes we got back as "seen" (even though we haven't necessarily seen them directly yet).
-                            // They will be pinged later in an attempt to verify them.
-                            packets::ResponseSpecific::FindNodeResponse(args) => {
-                                let mut state = state.lock().unwrap();
-                                for node in &args.nodes {
-                                    if node.id.is_valid_for_ip(&node.address.ip()) {
-                                        state.buckets.add_or_update(node.clone(), false);
-                                    }
+                        if let packets::ResponseSpecific::FindNodeResponse(args) = response_variant {
+                            let mut state = state.lock().unwrap();
+                            for node in &args.nodes {
+                                if node.id.is_valid_for_ip(&node.address.ip()) {
+                                    state.buckets.add_or_update(node.clone(), false);
```

Comment:
Please preserve the comment so I remember wtf I was thinking 😄 

```suggestion
                        // Special handling for find_node responses
                        // Add the nodes we got back as "seen" (even though we haven't necessarily seen them directly yet).
                        // They will be pinged later in an attempt to verify them.
                        if let packets::ResponseSpecific::FindNodeResponse(args) = response_variant {
                            let mut state = state.lock().unwrap();
                            for node in &args.nodes {
                                if node.id.is_valid_for_ip(&node.address.ip()) {
                                    state.buckets.add_or_update(node.clone(), false);
```

Reply:
Done